src/tz_android.rs

```diff
@@ -0,0 +1,24 @@
+use std::env;
+use android_system_properties::AndroidSystemProperties;
+
+// From https://android.googlesource.com/platform/ndk/+/android-4.2.2_r1.2/docs/system/libc/OVERVIEW.html
+// The name of the current timezone is taken from the TZ environment variable,
+// if defined. Otherwise, the system property named 'persist.sys.timezone' is
+// checked instead.
+
+const TIMEZONE_ENV_KEY: &str = "TZ";
+const TIMEZONE_PROP_KEY: &str = "persist.sys.timezone";
+
+pub(crate) fn get_timezone_inner() -> Result<String, crate::GetTimezoneError> {
+    return match env::var(TIMEZONE_ENV_KEY) {
```

Comment:
Actually, we don't check the environment variables for any other OS, and unless Android is **very** special in this regard, I don't think we should do it in here, either. Cf: https://github.com/time-rs/time/security/advisories/GHSA-wcg3-cvx6-7396

Reply:
I'm not sure about this. `TZ` environment is not set by default in Android, so let's assume it that `TZ` is always not set. I will remove this later.